include/lapack/gebd2.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef GEBD2
+#define GEBD2
+
+// #include <utility>
+// #include <plugins/tlapack_stdvector.hpp>
+// #include <plugins/tlapack_legacyArray.hpp>
+// #include <plugins/tlapack_debugutils.hpp>
+// #include <tlapack.hpp>
+
+#include "base/utils.hpp"
+#include "base/types.hpp"
+#include "lapack/larfg.hpp"
+#include "lapack/larf.hpp"
+
+/** 
+* hand-crafted  reduction to bidiagonal form of a general m-by-n matrix A, level 2, right looking algorithm
```

Comment:
These comments should be above your function definition

They are also not complete yet. Have a look at other <T>LAPACK function for examples of detailed documentation.
You can also copy documentation from [LAPACK](https://www.netlib.org/lapack/explore-html/d3/d01/group__complex16_g_ecomputational_gad5a784fc93cbf57b6b8ff3f0ff229cbe.html)